hubble/src/arb.rs

```diff
@@ -141,31 +153,28 @@ impl Arb {
             .rollup_finalization_config
             .l1_next_node_num_slot_offset_bytes
             .inner() as usize;
-
         let raw_slot = self
             .l1_client
             .get_storage_at(
-                ethers::types::H160::from(self.rollup_finalization_config.l1_contract_address),
-                ethers::types::H256(
+                alloy::primitives::Address::new(
+                    (self.rollup_finalization_config.l1_contract_address).0,
+                ),
+                alloy::primitives::Uint::from_be_bytes(
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/unionlabs/union/pull/2693#issuecomment-2293655644>#ef6b49</a><span> - 08/16/2024, 03:05 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The 'Address::new' function expects a byte array of length 20. If 'self.rollup_finalization_config.l1_contract_address.0' is not of the correct length, this will cause a runtime error.
</div>
<div id="fix">
<b>Fix:</b> Ensure that 'self.rollup_finalization_config.l1_contract_address.0' is a byte array of length 20 before passing it to 'Address::new'.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -159,7 +159,9 @@
             alloy::primitives::Address::new(
                 (self.rollup_finalization_config.l1_contract_address).0,
             ),
             alloy::primitives::Uint::from_be_bytes(
                 self.rollup_finalization_config
                     .l1_latest_confirmed_slot
                     .to_be_bytes(),
             ),
 +           if (self.rollup_finalization_config.l1_contract_address).0.len() != 20 {
 +               return Err(eyre!("Invalid address length"));
 +           }
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged